ads1110.h

```diff
@@ -19,7 +19,7 @@
 
 // define struct for AMS5915 sensor
 typedef struct {
-	float voltage_factor;
+	float voltage_factor[2];
```

Comment:
I'd just suggest naming these floats. E.g. have "voltage_factor" and "voltage_offset", instead of two factors.

Looks good otherwise.

Reply:
I agree with kedder. Please make two seperate float variables with eloquent names instead of an array.

Reply:
Okay, I can fix that.